### PR TITLE
[Handlers] Keep rustic count

### PR DESCRIPTION
**Pull request by tjoyal**

### CODE REVIEW

File: app/models/handler.rb [hunk 1]
```diff
@@ -23,6 +23,9 @@ def service
 
   def handle(payload)
     service.new(settings, payload).call
+
+    self.handle_count += 1
```
Comment (christianblais) [inline app/models/handler.rb:27]: This is basically `increment!(:handle_count)`. Although do we want to find a threadsafe way to do this too?

Comment (christianblais) [reply app/models/handler.rb:27]: https://apidock.com/rails/ActiveRecord/CounterCache/ClassMethods/increment_counter

Comment (tjoyal) [reply app/models/handler.rb:27]: increment_counter TIL.

> Although do we want to find a threadsafe way to do this too?

I'd say no for now. If we had a job infrastructure yet probably, but I'd rather missing few count increment than risking a deadlock for now?

It was more of a "we have nothing, let's have something even if it is more or less accurate"

Comment (tjoyal) [reply app/models/handler.rb:27]: Looks like `increment_counter` is threadsafe (not transaction save, which we do not need here)

**Pull request merged**